docker-compose_darwin.spec

```diff
@@ -0,0 +1,108 @@
+# -*- mode: python ; coding: utf-8 -*-
```

Comment:
I'm guessing there isn't a way we can avoid duplicating the information here vs in `docker-compose.spec`?